0000-template.md

```diff
@@ -0,0 +1,116 @@
+# Table of Contents
+- [Summary]
+- [Tracking Issues](#tracking-issue)
+  - [Discourse Discussion](#discourse-discussion)
+- [Motivation]
+- [Guide Level Explanation](#guide-level-explanation)
+- [Reference Level Explanation](#reference-level-explanation)
+- [Drawbacks]
+- [Rationale and Alternatives](#rationale-and-alternatives)
+- [Prior Art](#prior-art)
+- [Unresolved Questions](#unresolved-questions)
+
+# Basic Info
+[basic]: #basic-info
+- Feature Name: (fill me in with a unique ident, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- [Tracking Issue](#tracking-issue): (leave this empty)
+- [Discourse Link](#discourse-discussion): (if there is one)
+
+After filling out these fields, please proceed to fill out the remaining sections of this document.
+
+# Summary
+[summary]: #summary
+
+One paragraph explanation of the feature.
+
+# Tracking Issue
+[tracking-issue]: #tracking-issue
+<details>
+<summary>Info about Tracking Issues and how they work</summary>
+These are issues created on GitHub that track the implementation progress of the RFC. They link to related issues, have checkboxes, or other structures that make them more of a meta-issue than an issue for adding a specific feature or fixing a bug.
+</details>
+
+## Discourse Discussion
```

Comment:
Leaving this here means that we have to get the forums ready before this PR is merged. I would also like to see this renamed to call it the "Amethyst Community Forums" or something like that because people may not know what Discourse is. A link would also be appreciated.

Reply:
Renamed.